### DISCUSSION TITLE
Latest Executable

### DISCUSSION THREAD

**Discussion by Aldrin-John-Olaer-Manalansan** [Aldrin-John-Olaer-Manalansan/RA2YR-MIX-Unprotector-MASTER#2] [General]
v2.0.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Aldrin-John-Olaer-Manalansan/RA2YR-reMIXer/releases/tag/Latest-Executable'>Latest Executable</a>.</em>

**Comment by liyucheng990123** [comment]
I encountered a situation where LMD cannot be fixed during use. You can optimize this aspect in the future. Additionally, I would like to ask how to crack the problem if the title is encrypted instead of protected, such as when the first few digits of the hexadecimal editor display 00000 2

**Reply by Aldrin-John-Olaer-Manalansan** [reply]
Create your LMD issue ticket at the issue section. Make sure to give me the URL of the mix file. You can also privately send the file url to my email.

Regarding the Encrypted Mixes, I don't touch those bacause I don't know how their behaviour works. Commonly an encrypted mix file are normally unprotected and can be viewd by xcx mixer.

**Reply by Aldrin-John-Olaer-Manalansan** [reply]
> I encountered a situation where LMD cannot be fixed during use. You can optimize this aspect in the future.

It can't be helped if the LMD is entirely missing inside the MIX file. But in v4.0.0 I'm planning on adding a feature that prompts the user to specify all .ini files (example: rules##.ini, art##.ini, eva##.ini, sound##.ini) which this tool could use to determine all possible filenames. 

<br><br>

> Additionally, I would like to ask how to crack the problem if the title is encrypted instead of protected, such as when the first few digits of the hexadecimal editor display 00000 2

Encrypted Files aren't supported yet in the current version, but in v4.0.0 I plan on supporting encrypted/checksummed mix files.

Though the development time of v4.0.0 will depend on my free time and how Inspiried I would be to allocate time to it.
